warehouse/query-core/src/main/java/datawave/query/jexl/visitors/CaseSensitivityVisitor.java

```diff
@@ -15,9 +15,8 @@
 
 /**
  * Upper case all identifier nodes as our field names are always upper case
- * 
- * 
- * 
+ * <p>
+ * Example: (foo == 'bar') -> (FOO == 'bar')
```

Comment:
Have you done a build with `-Ddist` enabled? My guess is this won't pass. You probably need:
```
Example: {@code (foo == 'bar') -> (FOO == 'bar')}
```

Reply:
It definitely fails the -Ddist build. Fixed in latest commit.